pkg/osquery/tables/macos_software_update/recommended_updates_table_test.go

```diff
@@ -0,0 +1,24 @@
+//go:build darwin
+// +build darwin
+
+package macos_software_update
+
+import (
+	"context"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_generateRecommendedUpdatesHappyPath(t *testing.T) {
+	t.Parallel()
+	table := Table{logger: log.NewNopLogger()}
+	t.Run("HappyPath", func(t *testing.T) {
```

Comment:
You don't need `t.Run` here -- there's no subtest in play. You can run the tests from `Test_generateRecommendedUpdatesHappyPath` directly